httpclient/src/test/java/feign/httpclient/ApacheHttpClientTest.java

```diff
@@ -188,6 +188,20 @@ public void postWithSpacesInPath() throws IOException, InterruptedException {
                 .hasBody("foo");
     }
 
+    @Test
```

Comment:
probably you want a test closer to the code, too.. ex one in RequestTemplateTest